packages/env/package.json

```diff
@@ -39,7 +39,7 @@
 		"got": "^10.7.0",
 		"inquirer": "^7.1.0",
 		"js-yaml": "^3.13.1",
-		"nodegit": "^0.26.2",
+		"nodegit": "^0.27.0",
 		"ora": "^4.0.2",
```

Comment:
Should we add a note in the CHANGELOG file? Technically, it means we didn't have proper support for Node v14 before.

Reply:
My thought is we could create a single item to roll under a set of changes to support v14, for example if we're ready to update workflows and anywhere else. 9.3 just went out, so we can plan it all under 9.4

Reply:
We won't publish it until WordPress 5.6 is released so we can do it in steps :)

Reply:
Is there a reason to hold on publishing  `wp-env` until  5.6?  it's a dev dependency.

Reply:
We could cherry-pick commits with dev tools changes and publish to npm earlier, but someone would have to volunteer 😄 There are no other blockers.

Reply:
I'm still blocked by other stuff, if this proves to be the only thing preventing me from using node 14 + npm 7, I will update.

Reply:
> We could cherry-pick commits with dev tools changes and publish to npm earlier, but someone would have to volunteer 😄 There are no other blockers.

How would I go about doing this? It would be helpful to release all of the recent wp-env changes!